source/_docs/scripts/conditions.markdown

```diff
@@ -318,6 +318,15 @@ condition:
   after_offset: "-01:00:00"
 ```
 
+This is an example of 1 hour offset after sunset.
+
+```yaml
+condition:
+  - condition: sun
+    after: sunset
+    before: sunrise
+```
```

Comment:
The description doesn't match the actual YAML code?

Reply:
This description is for the YAML code above the the description as it follows the structure of the other examples in the document.

"condition:
  condition: sun
  after: sunset
  after_offset: "-01:00:00"
This is an example of 1 hour offset after sunset."